project/SbtLauncherPlugin.scala

```diff
@@ -0,0 +1,50 @@
+import sbt._
+import Keys._
+
+
+object SbtLauncherPlugin extends AutoPlugin {
```

Comment:
Is this plugin hard-coded to sbt usage? Can we make it general purpose and move it to sbt/launcher?